product_docs/docs/biganimal/release/administering_cluster/notifications.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Notifications 
+---
+
+BigAnimal supports receiving in-app and email notifications and allows you to choose receive notifications of a selected type. 
+
+Different types of events are sent as notifications. These notifications are set at different levels and users with different roles can configure this notifications. This table provides the list of events sent as notifications grouped by different levels at which they can be set:
+
+| Level        | Event                                                                                            | Role                             |
+|--------------|--------------------------------------------------------------------------------------------------|----------------------------------|
+| Organization | Payment method added                                                                             | Organization owner/admin         |
+| Organization | Personal access key is expiring                                                                  | Organization owner               |
+| Organization | Machine user access key is expiring                                                              | Organization owner               |
+| Project      | Upcoming maintenance upgrade on a cluster (24hr)                                                 | Project owner/editor             |
+| Project      | Successful maintenance upgrade on a cluster                                                      | Project owner/editor             |
+| Project      | Failed maintenance upgrade on a cluster                                                          | Project owner/editor             |
+| Project      | Paused cluster will automatically reactivated in 24 hours                                        | Project owner/editor             |
+| Project      | Paused cluster was automatically reactivated                                                     | Project owner/editor             |
+| Project      | You must set up the encryption key permission for your CMK-enabled cluster                       | Project owner/editor             |
+| Project      | Key error with CMK-enabled cluster                                                               | Project owner and project editor |
+| Project      | User is invited to a project (displays only to the Project owner)                                | Project owner                    |
+| Project      | New role is assigned to you                                                                      | Project user                     |
```

Comment:
`Project user` is ambiguous --> this event and L23's "Role is unassigned from you" event notify the user whose role assignment is changed, instead of "all project users".

Reply:
What can we mention here? Individual user?

Reply:
@nidhibhammar we group these notifications at `Account` level

![Screenshot 2024-04-05 at 4 39 08 PM](https://github.com/EnterpriseDB/docs/assets/23738352/876c608c-c758-41a6-b071-719a0e81335c)